### PR TITLE
Kamu UI 192 small style improvements

### PR DESCRIPTION
**Pull request by dmitriy-borzenko**

### CODE REVIEW

File: src/app/auth/account/account.component.spec.ts [hunk 1]
```diff
@@ -145,22 +145,20 @@ describe("AccountComponent", () => {
         expect(() => flush()).toThrow(new AccountNotFoundError());
     }));
 
-    it("should check activeTab when URL not exist query param tab", () => {
+    it("should check activeTab when URL not exist query param tab", fakeAsync(() => {
```
Comment (zaychenko-sergei) [inline src/app/auth/account/account.component.spec.ts:148]: Why `fakeAsync`? I don't see any ticks or flush

Comment (dmitriy-borzenko) [reply src/app/auth/account/account.component.spec.ts:148]: Removed

**Review by EvilAlexei — APPROVED** [review]

File: src/app/auth/account/account.component.html [hunk 2]
```diff
@@ -115,6 +118,7 @@ <h2 class="text-start pt-3 px-3 h6 organization">Organizations</h2>
                         value="Stars"
                         [checked]="activeTab === AccountTabs.STARS"
                         (click)="onSelectStarsTab(user)"
+                        disabled="true"
```
Comment (zaychenko-sergei) [inline src/app/auth/account/account.component.html:121]: I can see you've added these attributes, but the tabs are stil clickable for some reason

**Review by zaychenko-sergei — CHANGES_REQUESTED** [review]
The following requirement is not covered:

Use correct account avatar in data preview:
image

I can see Kamu picture in all configurations.

**Comment by dmitriy-borzenko** [comment]
> The following requirement is not covered:
> 
> Use correct account avatar in data preview: image
> 
> I can see Kamu picture in all configurations.

Fixed

File: src/app/components/lineage-graph/lineage-graph.component.ts [hunk 3]
```diff
@@ -63,4 +63,20 @@ export class LineageGraphComponent {
...
+        if (headerViewMenuElement && datasetTabsElement && headerElement) {
+            const headerHeight = headerElement.getBoundingClientRect().height;
+            const headerViewHeight = headerViewMenuElement.getBoundingClientRect().height;
+            const headerMarginTop = +getComputedStyle(headerViewMenuElement).marginTop.split("px")[0];
+            const datasetTabsElementHeight = datasetTabsElement.getBoundingClientRect().height;
+            return (
+                window.innerHeight - (headerHeight + headerViewHeight + headerMarginTop + 2 * datasetTabsElementHeight)
+            );
+        }
+        return screen.height - 390;
```
Comment (zaychenko-sergei) [inline src/app/components/lineage-graph/lineage-graph.component.ts:80]: It's architecturally incorrect for middle-layer component to assume a certain environment in higher level components in the hierarchy. It should be getting the computed number through @Input port. Only the main component `AppComponent` is aware of where header exists. Only `DatasetComponent` knows about dataset tabs. `LineageGraphComponent` is not allowed to access this information.

Comment (zaychenko-sergei) [reply src/app/components/lineage-graph/lineage-graph.component.ts:80]: Plz report a separate ticket for this problem, and revert to constant for now.

**Review by zaychenko-sergei — APPROVED** [review]

**Pull request merged**